### PR TITLE
Added missing box-sizing for the upload icon pseudoelement.

### PR DESCRIPTION
**Pull request by dkonopka**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Added missing box-sizing to the successful upload icon. Closes ckeditor/ckeditor5/issues/1095.

### CODE REVIEW

File: theme/ckeditor5-image/imageuploadicon.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	--ck-color-image-upload-icon: hsl(0, 0%, 100%);
 	--ck-color-image-upload-icon-background: hsl(120, 100%, 27%);
 
-	--ck-image-upload-icon-size: 1.25em;
+	--ck-image-upload-icon-size: 20px;
```
Comment (Reinmar) [inline theme/ckeditor5-image/imageuploadicon.css]: Why is this a variable?

Comment (Reinmar) [reply theme/ckeditor5-image/imageuploadicon.css]: And the below too.

Comment (Reinmar) [reply theme/ckeditor5-image/imageuploadicon.css]: cc @oleq 

Do we want to allow modifying even such styles via CSS vars?

Comment (dkonopka) [reply theme/ckeditor5-image/imageuploadicon.css]: `/* To make animation scalable. */` 😄 If someone decide that icon upload is too small then just one CSS Variable will change it and "tick" animation will be okey too. But we can get rid of this.

Comment (Reinmar) [reply theme/ckeditor5-image/imageuploadicon.css]: Can't we make it bigger so people don't have to scale it? :D

Comment (Reinmar) [reply theme/ckeditor5-image/imageuploadicon.css]: Jokes aside – it's a different story to allow easily adjusting the look of CKE5 to your app, so it fits well. And it's a different thing to allow tweaking the size of a self-contained thing like this icon.

Comment (oleq) [reply theme/ckeditor5-image/imageuploadicon.css]: TBH, I'm OK with it :P We use vars for similar purposes somewhere already so it's not a big deal.

**Pull request merged**